PWGHF/Core/SelectorCuts.h

```diff
@@ -357,40 +357,67 @@ static const std::vector<std::string> labelsCutVar = {"mK0s", "mLambda", "mGamma
 
 namespace hf_cuts_dplus_to_pi_k_pi
 {
-static const int nBinsPt = 12;
-static const int nCutVars = 8;
+static const int nBinsPt = 24;
+static const int nCutVars = 13;
 // default values for the pT bin edges (can be used to configure histogram axis)
 // offset by 1 from the bin numbers in cuts array
 constexpr double binsPt[nBinsPt + 1] = {
-  1.,
-  2.,
-  3.,
-  4.,
-  5.,
-  6.,
-  7.,
-  8.,
-  10.,
-  12.,
-  16.,
-  24.,
-  36.};
+    0.,
+    1.,
+    1.5,
+    2.,
+    2.5,
+    3.,
+    3.5,
+    4.,
+    4.5,
+    5.,
+    5.5,
+    6.,
+    6.5,
+    7.,
+    7.5,
+    8.,
+    9.,
+    10.,
+    11.,
+    12.,
+    14.,
+    16.,
+    24.,
+    36.,
+    50.
+
+  };
 auto vecBinsPt = std::vector<double>{binsPt, binsPt + nBinsPt + 1};
 
 // default values for the cuts
 // selections from pp at 5 TeV 2017 analysis https://alice-notes.web.cern.ch/node/808
```

Comment:
Please update the reference to the analysis note in which these cuts are used, it could be helpful to get details about the meaning of the variables.

Reply:
Thanks @fcatalan92 , i understand.. I'll implement and update...Thanks a lot....